src/include/duckdb/common/operator/cast_operators.hpp

```diff
@@ -27,13 +27,27 @@ struct TryCast {
 	}
 };
 
+struct TryCastCommaSeparated {
```

Comment:
This separate struct does not need to exist, no? Only `TryCastErrorMessageCommaSeparated` would be sufficient.